servers/lumen/app/Http/Controllers/DiagnosisController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Diagnosis;
+use Illuminate\Http\Request;
+use Ramsey\Uuid\Uuid;
+use DB;
+use Predis\Autoloader;
+
+class DiagnosisController extends Controller
+{
+    public function getAllDiagnosis()
+    {
+        return response()->json(Diagnosis::all());
```

Comment:
There should be get all the data from temporal DB.
Ref: reminderController.php

Reply:
I don't understand the temporal DB code for the reminder controller